packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -136,17 +142,73 @@ describe('HCP home page', () => {
+    await act(async () => {
+      render(getHomePage())
+    })
+
+    const secondaryBar = screen.getByTestId('patients-secondary-bar')
+    const addPatientButton = within(secondaryBar).getByRole('button', { name: 'Add patient' })
+    expect(addPatientButton).toBeInTheDocument()
+
+    addPatientButton.click()
+
+    const addPatientDialog = screen.getByRole('dialog')
+    expect(addPatientDialog).toBeInTheDocument()
+
+    const title = within(addPatientDialog).getByText('New patient')
+    expect(title).toBeInTheDocument()
+
+    const warningLine1 = within(addPatientDialog).getByText('By inviting this patient to share their data with me and their care team, I declare under my professional responsibility that I am part of this patient’s care team and, as such, have the right to access the patient’s personal data according to the applicable regulations.')
+    expect(warningLine1).toBeInTheDocument()
+
+    const warningLine2 = within(addPatientDialog).getByTestId('modal-add-patient-warning-line2')
+    expect(warningLine2).toHaveTextContent('Read our Terms of use and Privacy Policy.')
+    const termsOfUseLink = within(addPatientDialog).getByRole('link', { name: 'Terms of use' })
+    expect(termsOfUseLink).toBeInTheDocument()
+    const privacyPolicyLink = within(addPatientDialog).getByRole('link', { name: 'Privacy Policy' })
+    expect(privacyPolicyLink).toBeInTheDocument()
+
+    const cancelButton = within(addPatientDialog).getByText('Cancel')
+    expect(cancelButton).toBeInTheDocument()
+
+    const invitePatientButton = within(addPatientDialog).getByRole('button', { name: 'Invite' })
+    expect(invitePatientButton).toBeInTheDocument()
+
+    const emailInput = within(addPatientDialog).getByRole('textbox', { name: 'Email' })
+    expect(emailInput).toBeInTheDocument()
+    fireEvent.change(emailInput, { target: { value: monitoredPatient.email } })
+
+    const select = within(addPatientDialog).getByTestId('patient-team-selector')
+    fireEvent.mouseDown(within(select).getByRole('button'))
+    screen.getByRole('listbox')
+    fireEvent.click(screen.getByRole('option', { name: teamTwo.name }))
+
+    const alreadyInTeamErrorMessage = within(addPatientDialog).getByText('This patient already shared their data with the team.')
+    expect(alreadyInTeamErrorMessage).toBeInTheDocument()
+    expect(invitePatientButton).not.toBeEnabled()
+
+    fireEvent.change(emailInput, { target: { value: pendingPatient.email } })
+    fireEvent.mouseDown(within(select).getByRole('button'))
+    screen.getByRole('listbox')
+    fireEvent.click(screen.getByRole('option', { name: teamThree.name }))
+
+    const pendingErrorMessage = within(addPatientDialog).getByText('This patient has already been invited and hasn\'t confirmed yet.')
+    expect(pendingErrorMessage).toBeInTheDocument()
+    expect(invitePatientButton).not.toBeEnabled()
```

Comment:
```suggestion
    expect(invitePatientButton).toBeDisabled()
```

Reply:
I added the check at the end of the test, I agree I think it's better now!